tests/client_tests/rust/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "rust-cratedb-tests"
+version = "0.1.0"
+authors = ["Mathias Fussenegger <mathias@crate.io>"]
+edition = "2018"
```

Comment:
not this year? ;)

Reply:
This was generated by `cargo new`, not sure why it took last year :) 

Will change it.

Reply:
Edition is actually cargo specific.
```
Caused by:
  failed to parse the `edition` key

Caused by:
  supported edition values are `2015` or `2018`, but `2019` is unknown
```